### PR TITLE
Document how to update the website on the server

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,51 @@ will be able to advise on further steps.
...
+server, in the directory `~/www.gap-system.org/Manuals` (so be very careful
+never to delete that!). To get them to appear in the right place on the
+website (they don't automatically, as they are not in the `~/www.gap-system.org/_site`
+directory), we configured nginx to put them there, via
+`/host/gap-web/nginx.d/www.gap-system.org/_global.conf`
+
+
+## Tweaking the server config
+
+In general, config files for the nging web server are in `/host/gap-web/nginx.d/`
```
Comment (wilfwilson) [inline README.md:57]:
```suggestion
In general, config files for the nginx web server are in `/host/gap-web/nginx.d/`
```

File: README.md [hunk 2]
```diff
@@ -21,3 +21,51 @@ will be able to advise on further steps.
...
+3. run `git up` (an alias we configured there for `git pull --ff-only`)
+
+4. run `jekyll build`
+
+That's it.
+
+
+## Warning about package manuals
+
+Note that `Manuals` directory is not in this git repository, as it is too
```
Comment (wilfwilson) [inline README.md:46]:
```suggestion
Note that the `Manuals` directory is not in this git repository, as it is too
```

**Review by wilfwilson — APPROVED** [review]
Thanks @fingolfin. I'm going to work out how I get access to the web server...

**Comment by fingolfin** [comment]
Thanks for the fixes!

**Pull request merged**